serverless/develop/serverless-autoscaling-developer.adoc

```diff
@@ -0,0 +1,98 @@
+[id="serverless-autoscaling-developer"]
+= Autoscaling
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: serverless-autoscaling-developer
+
+toc::[]
+
+Knative Serving provides automatic scaling, or _autoscaling_, for applications to match incoming demand. For example, if an application is receiving no traffic, and scale-to-zero is enabled, Knative Serving scales the application down to zero replicas. If scale-to-zero is disabled, the application is scaled down to the xref:../../serverless/develop/serverless-autoscaling-developer.adoc#serverless-autoscaling-developer-minscale[minimum number of replicas specified for applications on the cluster]. Replicas can also be scaled up to meet demand if traffic to the application increases.
+
+If autoscaling is enabled for your cluster, you can configure concurrency and scale bounds for your application.
```

Comment:
> If autoscaling is enabled for your cluster

Let's make it more specific as there is [OCP autoscaling too](https://docs.openshift.com/container-platform/4.7/machine_management/applying-autoscaling.html). How about: "If autoscaling is enabled in Serverless configuration"

Reply:
Updated to "Knative autoscaling"